winnativeeventfilter.cpp

```diff
@@ -1309,6 +1309,9 @@ bool WinNativeEventFilter::nativeEventFilter(const QByteArray &eventType,
                     QPointF point = {obj->property("x").toReal(), obj->property("y").toReal()};
                     for (QObject *parent = obj->parent(); parent; parent = parent->parent()) {
                         point += {parent->property("x").toReal(), parent->property("y").toReal()};
+                        if(parent->isWindowType()) {
```

Comment:
Adding a space between `if` and `(` would be much better.

Reply:
Done in f43792a. Need to get astyle set up to avoid tweaks like this :)

Reply:
Yes you are right. Thanks!